cypress/support/step_definitions/index.ts

```diff
@@ -58,19 +67,45 @@ When('I wait for {int} milliseconds/ms', function (ms: number) {
   cy.wait(ms)
 })
 
-When(/^I "(.*)"(.*)? on the "(.*)" element$/, (event: string, object: string | number | undefined, selector: string) => {
+When(/^I "(.*)" the "(.*)" element(?: and select "(.*)".*)?$/, (event: string, selector: string, value?: string) => {
```

Comment:
Just to note here our discussion form yesterday (no action here just noting).

We discussed the fact that I found out that you can't `"click" on the select` and `then "click" on the option` with Cypress, so I originally went with a "select" verb so it you can just replace "click" with "select".

We chatted through this potentially being weird due to any differences between a native select and a custom select, where probably with a custom select you would be clicking to open and then clicking to select. but as and when that becomes something we need to do we can hide the difference in the step.

We also chatted through the tiny nit-like `on` removal which was there to make all triggers/events read nice i.e:

```
When I "click" the "button" element
When I "hover" the "button" element 
vs
When I "click" on the "button" element
When I "hover" on the "button" element
```
"hover on" reading nicer than just "hover"

Bit of a nit so I'm happy either way really, but thats why the `on` was originally there. No change necessary for approval here.